test/Nancy.Tests/Unit/DefaultNancyBootstrapperFixture.cs

```diff
@@ -70,6 +70,7 @@ public void Request_should_be_available_to_request_startup()
             this.bootstrapper.RequestStartupLastRequest.ShouldBeSameAs(request);
         }
 
+#if !CORE

```

Comment:
This was mostly laziness/expediency. I just haven't found the dynamic generator for the new framework yet.